packages/foreman/foreman/foreman.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Foreman
+Documentation=https://theforeman.org
+After=network.target remote-fs.target nss-lookup.target
+
+[Service]
+Type=forking
+User=foreman
+TimeoutSec=90
+WorkingDirectory=/usr/share/foreman
+ExecStart=/usr/sbin/foreman
```

Comment:
Is it not possible to inline `scl enable tfm "rails server -e $FOREMAN_ENV -p $FOREMAN_PORT"` here?

Reply:
No -- you need a full path or systemd complains. Also, trying ot using `/usr/bin/tfm-ruby` just failed hard for me.

Reply:
And `/usr/bin/scl ...`?